### PR TITLE
Point artifactory in sandbox to new domain

### PR DESCRIPTION
**Pull request by timja**
As part of tactical decommissioning we need to remove the proxyin server that this service is using,
domain name needs to change as the ingress controller is only setup with .sandbox.platform.hmcts.net atm and it makes sense to change it

### CODE REVIEW

**Comment by timja** [comment]
Need to debug locally, will do tomorrow:

> sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

Probably an issue with the cert chain

**Review by naikajah — DISMISSED** [review]

**Comment by timja** [comment]
works in browser FTR

**Comment by timja** [comment]
I've added the chain in https://github.com/hmcts/cnp-flux-config/pull/6424, curl now works from a linux VM, previously it complained about the issuer.

It worked on my mac for everything without the change 👅 , browser, curl, java...

File: Jenkinsfile_pipeline_test [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!groovy
-@Library('Infrastructure') _
+@Library('Infrastructure@repoint-sandbox-artifactory') _
```
Comment (timja) [inline Jenkinsfile_pipeline_test:2]:
```suggestion
@Library('Infrastructure') _
```

**Pull request merged**